src/widgets/_index.scss

```diff
@@ -18,28 +18,35 @@ messagedialog,
 @define-color success_color @LIME_700;
 @define-color warning_color @BANANA_900;
 
+@define-color selected_bg_color #{"alpha(@accent_color_100, 0.3)"};
+@define-color selected_fg_color #{"@accent_color_900"};
+
 // Things Dazzle expects
 @define-color borders #{"" + $toplevel-border-color};
 @define-color content_view_bg #{""+ bg_color(2)};
 @define-color theme_fg_color #{"" + $fg-color};
-@define-color theme_selected_fg_color #{"" + $fg-color};
+@define-color theme_selected_bg_color #{"@selected_bg_color"};
+@define-color theme_selected_fg_color #{"@selected_fg_color"};
 @define-color wm_shadow #{rgba(black, 0.2)};
 
-@define-color selected_bg_color #{'alpha(@accent_color_100, 0.3)'};
-@define-color selected_fg_color #{'@accent_color_900'};
+selection {
+    @extend %selected_item;
+}
+
+.gtkstyle-fallback {
```

Comment:
Can you make a comment about what this is and where it's used

Reply:
This is used wherever an item/widget uses GTK colors but isn't GTK proper.
Very useful when dealing with non elementary apps.

Reply:
In another PR from this, I'll add in non-selected states to it to complete it.